release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -967,6 +967,16 @@ With this release, replacement control plane nodes are assigned to the correct i
 [id="ocp-4-13-installer-bug-fixes"]
 ==== Installer
 
+* Previously, DNS records that the installation program created were not removed when uninstalling a private cluster. With this update, the installation program now correctly removes these DNS records. (link:https://issues.redhat.com/browse/OCPBUGS-7973[OCPBUGS-7973]
+
+* Previously, the bare metal installer-provisioned infrastrature used port 80 for providing images to the Baseboard Management Controller (BMC) and the deployment agent. Security risks could exist with port 80, because this port is commonly chosen for internet communcations. The bare metal installer-provisioned infrastrature now used port 6180 for serving images that are used by the `metal3` pod on deployed clusters. (link:https://issues.redhat.com/browse/OCPBUGS-8511[*OCPBUGS-8511*])
```

Comment:
```suggestion
* Previously, the bare metal installer-provisioned infrastructure used port 80 for providing images to the Baseboard Management Controller (BMC) and the deployment agent. Security risks could exist with port 80, because this port is commonly chosen for internet communications. The bare metal installer-provisioned infrastructure now uses port 6180 for serving images that are used by the `metal3` pod on deployed clusters. (link:https://issues.redhat.com/browse/OCPBUGS-8511[*OCPBUGS-8511*])
```

Reply:
I took the note verbatim. Well spotted. I've learned to read over them in detail before creating a PR. Thanks!